CAIPs/caip-2.md

```diff
@@ -0,0 +1,110 @@
+---
+caip: 2
+title: Blockchain ID
```

Comment:
Blockchain chain_id format

Reply:
I'd try to keep the technical variable name `chain_id` out of a title. Is "format" the right word here? What about "Blockchain ID specification"?

Reply:
Agreed, specification is a better name than format. Let's call it "Blockchain ID Specification"

Reply:
Implemented in https://github.com/ChainAgnostic/CAIPs/pull/9/commits/84431bc7f77c2c5bb4a8496c65d94557a9223fd8. I assume we use the headline style that capitalize (almost) every word then. Resolving this for now, feel free to re-open.